cmd/main.go

```diff
@@ -20,6 +21,12 @@ func main() {
 	// Set logging output to standard console out
 	log.SetOutput(os.Stdout)
 
+	// kubernetes client-go uses klog, which logs to file by default. Change defaults to log to stderr instead of file.
+	klogFlags := flag.NewFlagSet("klog", flag.ExitOnError)
```

Comment:
do you think it makes sense to also configure `glog` like in this example https://github.com/kubernetes/klog/blob/master/examples/coexist_glog/coexist_glog.go?

Reply:
Don't really think it would be beneficial, although it probably doesn't hurt either, reasoning:
- the glog package doesn't log to disk by default, that's something specific in the klog package; and that is the only behaviour which actually gets override here
- the project isn't using the levelled glog package but "plain" log; and I also don't see glog being used in any of the dependencies?

Reply:
that's sound good. thanks